grakn-graql/src/main/java/ai/grakn/graql/internal/query/analytics/PathQueryImpl.java

```diff
@@ -62,7 +63,8 @@
         ComputerResult result;
         try {
             result = getGraphComputer().compute(
-                    new ShortestPathVertexProgram(subTypeNames, sourceId, destinationId),
+                    //TODO: Look into passing sourceId and destinationId as ConceptId. Not possible right now because it's not serializable
+                    new ShortestPathVertexProgram(subTypeNames, sourceId.getValue(), destinationId.getValue()),
```

Comment:
My understanding of serialisable isn't very good, but I think this is as easy as just `ConceptId implements Serializable`.